Simplified/Book/UI/NYPLBookButtonsView.m

```diff
@@ -178,24 +169,16 @@ - (void)updateButtons
       break;
     case NYPLBookButtonsStateDownloadNeeded:
     {
+      NSString *title = (self.book.defaultAcquisitionIfOpenAccess || !NYPLUserAccount.sharedAccount.authDefinition.needsAuth) ? NSLocalizedString(@"Delete", nil) : NSLocalizedString(@"Return", nil);
```

Comment:
these are just indentation changes because I remove the `if (self.showReturnButtonIfApplicable) ...`